### PR TITLE
Update de.lua missing keys

### PR DESCRIPTION
**Pull request by bluecraank**
**Describe Pull request**
First, make sure you've read and are following the contribution guidelines and style guide and your code reflects that.
Write up a clear and concise description of what your pull request adds or fixes and if it's an added feature explain why you think it should be included in the core.

Some corrections for German :)

If your PR is to fix an issue mention that issue here

**Questions (please complete the following information):**
- Have you personally loaded this code into an updated qbcore project and checked all it's functionality? [yes/no] (Be honest) y
- Does your code fit the style guidelines? [yes/no] y
- Does your PR fit the contribution guidelines? [yes/no] y

### CODE REVIEW

File: locales/de.lua [hunk 1]
```diff
@@ -23,9 +23,12 @@ local Translations = {
         ["heading_copied"] = "Heading in Zwischenablage kopiert!",
         ["changed_perm"] = "Gruppenrechte geändert",
         ["entered_vehicle"] = "Ins Fahrzeug gestiegen",
-        ["success_vehicle_owner"] = "Das Fahrzeug ist nicht deins!",
+        ["success_vehicle_owner"] = "Das Fahrzeug ist nun deins!",
+        ["success_entered_vehicle"] = "Ins Fahrzeug teleportiert"
         ["receive_reports"] = "Du erhälst Reports",
         ["entity_copy"] = "Freeaim-Entitätsinfo in die Zwischenablage kopiert!",
+        ["noclip_enabled"] = "NoClip aktiviert",
+        ["noclip_disabled"] = "NoClip deaktiviert"
```
Comment (GodyFromDiscord) [inline locales/de.lua:31]: Missing a comma

File: locales/de.lua [hunk 2]
```diff
@@ -149,12 +152,16 @@ local Translations = {
         ["entity_view_vehicles"] = "Fahrzeuge anzeigen",
         ["entity_view_objects"] = "Objekte anzeigen",
         ["entity_view_freeaim_copy"] = "Freeaim-Entitätsinformationen kopieren",
+		["spawn_weapons"] = "Spawne Waffen",
+		["max_mods"] = "Full Tuned",
+        ["hud_dev_mode"] = "HUD Entw. Modus"
```
Comment (GodyFromDiscord) [inline locales/de.lua:157]: Missing a comma

File: locales/de.lua [hunk 3]
```diff
@@ -189,6 +196,8 @@ local Translations = {
         ["entity_view_vehicles_desc"] = "Fahrzeuginformationen in der Welt aktivieren/deaktivieren",
         ["entity_view_objects_desc"] = "Objektinformationen in der Welt aktivieren/deaktivieren",
         ["entity_view_freeaim_copy_desc"] = "Kopiert die Free Aim Entitätsinfo in die Zwischenablage",
+        ["spawn_weapons_desc"] = "Spawne eine Waffe",
+        ["hud_dev_mode_desc"] = "HUD Entw. Modus ein/aus"
```
Comment (GodyFromDiscord) [inline locales/de.lua:200]: Missing a comma

File: locales/de.lua [hunk 4]
```diff
@@ -261,10 +270,7 @@ local Translations = {
...
 }
 
-if GetConvar('qb_locale', 'en') == 'de' then
-    Lang = Locale:new({
-        phrases = Translations,
-        warnOnMissing = true,
-        fallbackLang = Lang,
-    })
-end
+Lang = Locale:new({
```
Comment (GodyFromDiscord) [inline locales/de.lua:273]: Why was this changed? You removed the part that checks the locale.

**Comment by ChristianBDev** [comment]
@bluecraank Please update the code to the requested changes so we can continue
